index.html

```diff
@@ -96,7 +96,7 @@ <h1 id="heroH1" aria-label="#Chat Saves TD">
             <span aria-hidden="true">TD</span>
           </h1>
           <p id="hero-caption">
-            Join us on <mark class="mark-teal">April 13, 2023</mark> for a
+            Join us on <mark class="mark-teal">May 1-5, 2023</mark> for a
```

Comment:
Everything you updated looks great, though I did notice a couple of other content updates we might want to make just to make so the site is overall easier to read and uses consistent language:

In The Mission, Debriefing section:
- [ ] initial `<p>` with the description, second sentence: lowercase "hackathon to be consistent throughout the site
- [ ] second `<li>`, lowercase "hackathon" in the `<p>` to be consistent throughout the site

In the Rules section:
- [ ] first `<p>`, second sentence: add an 's' to the end of 'role' so it reads "Each team will include a mix of roles."

In the FAQ section:
- [ ] second `<li>`, `<h3>`: lowercase "hackathon" so it matches the other instances in the FAQs
- [ ] third `<li>`, first sentence of the `<p>`: change this so the tenses match to "Our main goal is to foster enthusiasm, experimentation, and learning."
- [ ] fourth `<li>`, third sentence of the `<p>`: add this link around the "page for that in our Confluence space" (or however you want to break that up): https://willowtree.atlassian.net/l/cp/wVvo8sDP
- [ ] last `<li>`, last sentence of the `<p>`: add the word "scheduling" between "A DRI" so our messaging is consistent with the second bullet point.

In the Registration section:
- [ ] lowercase "hackathon" in the first sentence of the `<p>`

Reply:
Good catch on these! :-)